README.md

```diff
@@ -45,18 +45,18 @@ following URLs:
 
 We considered both [Consul](https://github.com/consul/consul) and Decidim,
 but were strongly recommended by the [The Participatory Budgeting Project](https://www.participatorybudgeting.org/mission/),
-a nonprofit that encourages PB processes globally, to use Decidim, as it 
```

Comment:
These whitespace diffs are because I have the `files.trimTrailingWhitespace` VS Code setting enabled. I can turn it off if we don't want to see these changes happen, or we can all embrace it :)